packages/zoe/src/zoe.js

```diff
@@ -760,8 +777,9 @@ const makeZoe = (additionalEndowments = {}) => {
       },
 
       isOfferActive: offerTable.isOfferActive,
-      getOffers: offerTable.getOffers,
-      getOffer: offerTable.get,
+      getOffers: offerHandles =>
+        offerTable.getOffers(offerHandles).map(removeNotifier),
+      getOffer: offerHandle => removeNotifier(offerTable.get(offerHandle)),
```

Comment:
I see that it's not a new policy, and so maybe better a question for @katelynsills 👍 

Why is it ok to expose the amount on the zoe service and not zcf?

With the allocations filtered out, the contract (via zcf) cannot see the allocations on the offers and uses a separate `getAllocations` API. I like that because it separates the things that can and cannot change about an offer.

Here, using the zoe service (which the contract can do as well) we see the full offers. Since the only thing that changes is the allocation, the notifier only provides notifications carrying only the allocations. This suggests that we should provide the same separation here.

Reply:
I think we just didn't have a reason to get the allocation separately before in the zoe service. I'm fine with splitting the zoe service getOffer into two methods: getOffer(s) and `getCurrentAllocation`(s)

Reply:
The fix to `getOffer`/`getOffers` looks good. What about `getCurrentAllocation`?

Reply:
I created issue #951